logging/config/cluster-logging-curator.adoc

```diff
@@ -5,14 +5,20 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} {product-version}, log curation is performed by Elasticsearch Curator, based on a xref:../../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[retention policy] that you define.
+You can configure log retention time. That is, you can specify how long the default Elasticsearch log store keeps indices by configuring a separate retention policy for each of the three log sources: infrastructure logs, application logs, and audit logs. For instructions, see xref:../../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[Configuring log retention time].
 
-The Elasticsearch Curator tool removes Elasticsearch indices that use the data model prior to {product-title} 4.5. You can modify the Curator index retention policy for your old data.
+[NOTE]
+====
+Configuring log retention time is recommended method for curating log data: It works with both the current data model and the previous data model from {product-title} 4.4 and earlier.
+====
+
+Optionally, to remove Elasticsearch indices that use the data model from {product-title} 4.4 and earlier, you can also use the Elasticsearch Curator. The following sections explain how to use the Elasticsearch Curator.
```

Comment:
I think it reads fine without "optionally". Your choice to keep it or leave it. I don't think it's a big deal either way. I brought it up because I know some writers try to avoid adverbs.

Reply:
Thanks, Jake. In this case, I think I'll keep it to help contrast this method with the preceding method. I'll be removing all the Elasticsearch Curator content in Logging 5.1 (OpenShift 4.8) so this will go away then.

Reply:
Sounds good!